app/views/schools/on_boarding/descriptions/_form.html.erb

```diff
@@ -1,9 +1,9 @@
-<%- self.page_title = 'Enter a short description of your school' -%>
+<%- self.page_title = 'Write a short description of your school' -%>
 
 <%
   self.breadcrumbs = {
     @current_school.name => schools_dashboard_path,
-    'Enter a short description of your school' => nil
+    'Write a short description of your school' => nil
```

Comment:
See previous comment about this page (we should make the appropriate content changes along with the other 'guidance for schools' recommendations, with dev help).